### PR TITLE
feat: Added solution for 06_if_else exercise

### LINKED ISSUE #15: Add solution: 06_if_else
Goal: Implement the solution for exercise "$ex".

Acceptance criteria:
- [ ] Implement code in internal/exercises/templates/$ex
- [ ] All tests pass for $ex (go test ./...)
- [ ] Add/adjust docs or comments if needed
- [ ] Cross-link this issue in the PR description

Notes:
- Keep code idiomatic and simple.
- Include edge cases and table-driven tests where relevant.

### PR DESCRIPTION
**Pull request by bdeb1337**
## Summary

Implemented the Sign function as solution to the if_else exercise (06_if_else)

## Checklist

- [x] Tests pass: `make verify` or `golearn verify <slug>`
- [x] Docs updated (README/CONTRIBUTING) if needed
- [x] No large new dependencies

## Screenshots / Output (if CLI UX)

image

## Related issues

Fixes #15 

## Summary by CodeRabbit

* **New Features**
  * Adds a number sign classification utility that returns "negative", "zero", or "positive" for integers — a lightweight, dependency-free way to label numeric values for display or conditional logic.

### CODE REVIEW

File: internal/exercises/solutions/06_if_else/if_else.go [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+// Returns "negative", "zero", or "positive" based on n
+func Sign(n int) string {
+	if n < 0 {
+		return "negative"
+	} else if n == 0 {
+		return "zero"
+	} else {
+		return "positive"
+	}
+}
```
[the remark this answers is not in the document]
Comment (bdeb1337) [reply internal/exercises/solutions/06_if_else/if_else.go:12]: Thank you, but I think it's in the right place: all solutions go under the `internal\exercises\solutions\$ex`... Take a look at other finished implementations.

**Comment by bdeb1337** [comment]
I actually like the diff that coderabbitai proposed in the nitpick comments:


`5-11`: **Remove redundant else clauses for idiomatic Go.**

After a return statement, else clauses are unnecessary in Go. Removing them results in flatter, more idiomatic code, which aligns with the PR objective to keep code "idiomatic and simple."

Apply this diff:

```diff
 func Sign(n int) string {
 	if n < 0 {
 		return "negative"
-	} else if n == 0 {
+	}
+	if n == 0 {
 		return "zero"
-	} else {
-		return "positive"
 	}
+	return "positive"
 }
 ```

But that would remove "demonstrating" `if else` and `else`. I would like to hear your opinion about what would be the better approach. I chose the more verbose way to illustrate `if`,`if else` and `else` deliberately.

Let me know what you think. :)

Kind regards.

**Review by zhravan — APPROVED** [review]
PR LGTM
